docs/platform/manager/using-the-linode-graphical-shell-glish/index.md

```diff
@@ -37,19 +38,17 @@ The process for launching a getty may differ depending on your distribution.
 
 ## Access Glish
 
-1.  From your Linode Dashboard, click on the **Remote Access** Tab. Under **Console Access**, click on the "Launch Graphical Web Console" link:
+1.  From your Linode Dashboard, select the Linode you want to access.
 
-    [![Glish access link.](glish-link_small.png)](glish-link.png)
+2.  Click the **more options ellipsis** corresponding to the Linode and select **Launch Console**.
 
-    A new window will pop up:
+3.  Select the **Glish** tab in the pop-up window that appears.
```

Comment:
@Rajakavitha1 A screenshot of the pop up window displaying Glish would be nice here.

Reply:
will do!!!